### PR TITLE
#1097 Use material UI to create hdfs sink component

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1097 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### Before merging
- [x] Fix unit tests

### CODE REVIEW

**Comment by konekoya** [comment]
@wu87988622 Please help me review this PR as the M28 is due on next Monday, let's speed up a bit...

File: ohara-manager/client/src/components/pages/PipelinePage/Connectors/TestConnectionBtn.js [hunk 1]
```diff
@@ -27,10 +27,10 @@ const TestConnectionBtn = props => {
     <FormGroup>
       <Button
```
Comment (wu87988622) [inline ohara-manager/client/src/components/pages/PipelinePage/Connectors/TestConnectionBtn.js]: Maybe change to Mui button?

Comment (konekoya) [reply ohara-manager/client/src/components/pages/PipelinePage/Connectors/TestConnectionBtn.js]: I'll do this in another PR :)

**Comment by konekoya** [comment]
Thanks for the review :)

**Pull request merged**